corehq/apps/hqwebapp/templatetags/menu_tags.py

```diff
@@ -159,3 +160,14 @@ def aliased_language_name(lang_code):
             if code == lang_code:
                 return name
         raise KeyError('Unknown language code %s' % lang_code)
+
+
+@register.simple_tag(takes_context=True)
+def prelogin_url(context, urlname):
+    """
+    A prefix aware url tag replacement for prelogin URLs
+    """
+    if context.get('url_uses_prefix', False):
+        return reverse(urlname, args=[context['LANGUAGE_CODE']])
```

Comment:
remind me where `LANGUAGE_CODE` gets in the context? should we consider also checking if it's present so this doesn't crash if it's not?